docs/apppromote.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: apppromote
+spec:
+  resources:
+    inputs:
+    - name: git-source
+      type: git
+  params:
+  - name: github-secret
+    type: string
+    description: seret name of the github that has the access token. The key name is token.
+  - name: from
+    type: string
+    description: github repository url to be promoted from
+    default: /workspace/git-source
+  - name: to
+    type: string
+    description: github repository url to be promoted to
+  - name: service
+    type: string
+    description: service name to be promoted
+  - name: github-config
+    type: string
+    description: configmap name of the gitconfig file that has user name, user e-mail.  The key name is gitconfig. It can be created by "kubectl create configmap <configmap name> --from-file=gitconfig -n promote"
```

Comment:
```suggestion
    description: configmap name of the gitconfig file that has user name and e-mail.  The key name is gitconfig. It can be created by "kubectl create configmap <configmap name> --from-file=$HOME/.gitconfig -n <namespace>"
```

Reply:
I fix it.